proposals/lip-0053.md

```diff
@@ -54,23 +54,22 @@ The `inboxUpdate` contains the information relative to the messages to be includ
 
 An array of cross-chain messages. See [LIP 0049][lip-0049] for the general properties and processing of cross-chain messages. The `crossChainMessages` property must contain consecutive messages from the outbox.
 
-##### messageWitness
+##### messageWitnessHashes
```

Comment:
The explanations of each property are usually part of the specifications as a bullet list directly after introducing the params schema. Introducing them here without any context seems odd and explaining properties of the specifications is also not in scope of the rationale.

Reply:
As mentioned, I would prefer to limit the changes to what is necessary for the protocol

Reply:
Okay, let us keep it.